### DISCUSSION TITLE
I want my old feed back.

### DISCUSSION THREAD

**Discussion by Jak2k** [community/community#66271] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I want the chronological feed back ASAP. Otherwise GitHub becomes unusable for me.

**Comment by Itumadavid1** [comment]
I understand your concern about wanting to have a chronological feed on GitHub. However, GitHub periodically updates its user interface and features, and some changes might not be reversible by users.

To express your feedback and request the return of a chronological feed, you can contact GitHub directly through their support channels. Here's how you can do it:

GitHub Support: You can contact GitHub Support directly through their [contact form](https://support.github.com/contact). Explain your issue and the importance of having a chronological feed for your workflow.

GitHub Community Forum: GitHub has an active [Community Forum](https://github.community/) where you can post your feedback and concerns. GitHub staff and other community members often engage in discussions there.

GitHub Twitter or Social Media: You can also reach out to GitHub on social media platforms like Twitter and express your concerns. They might respond to direct messages or tweets.

GitHub Feedback Page: GitHub has a feedback page where you can submit your suggestions and feedback regarding the platform. You can access it at https://github.com/github/feedback.

When providing feedback, it's essential to be clear and concise about your needs and how the changes have impacted your workflow. While GitHub does make updates, they also value user feedback and may consider adjustments based on user input.

Remember that software platforms like GitHub often evolve, and changes may not always align with every user's preferences. Expressing your concerns directly to GitHub is the best way to advocate for the features you find essential.

**Reply by Sec-ant** [reply]
Spamming the discussion with AI generated content doesn't help.

**Comment by fogfish** [comment]
New Home Pages does not provide any value for those who does not have many followers, activities, etc. The old approach had more value. With the new interface, I would change the landing page to be github.com/:username

**Reply by hlerenow** [reply]
I have get a same problem

**Comment by wangrongding** [comment]
# Github Old Feed  

💪 Using this script you can restore the old version of the feed.
https://github.com/wangrongding/github-old-feed  

<table>
  <tr>
      <td width="50%" align="center"><b>Before</b></td>
      <td width="50%" align="center"><b>After use</b></td>
  </tr>
  <tr>
     <td><img src="https://assets.fedtop.com/picbed/1694066442147.jpg"/></td>
     <td><img src="https://assets.fedtop.com/picbed/1694066437621.jpg"/></td>
  </tr>
</table>

**Reply by hlerenow** [reply]
That's cool

**Comment by ghost** [comment]
And this new UI is really slow btw

**Comment by lloydjatkinson** [comment]
This is classic incompetent product owners making change for the sake of change.

**Comment by Jak2k** [comment]
Let's make [our own fronted](https://github.com/Jak2k/Alternative-GitHub-UI/discussions) for GitHub to be more independent from it's UI changes.
